templates/articles/_comment.html

```diff
@@ -0,0 +1,27 @@
+<turbo-frame id="comment-{{ comment.pk }}">
+<div class="card">
+    <div class="card-block">
+        <p class="card-text">
+            {{ comment.body }}
+        </p>
+    </div>
+    <div class="card-footer">
+        <a href="" class="comment-author">
```

Comment:
Here's is missing the link as well:
```suggestion
        <a href="{% url 'profile_view' comment.author.user.username %}" class="comment-author">
```